code/game/sound.dm

```diff
@@ -1,4 +1,22 @@
-/proc/playsound(atom/source, soundin, vol as num, vary, extrarange as num, falloff, frequency = null, channel = 0, pressure_affected = TRUE, ignore_walls = TRUE)
+/*! playsound
+
+playsound is a proc used to play a 3D sound in a specific range. This uses SOUND_RANGE + extra_range to determine that.
+
+source - Origin of sound
+soundin - Either a file, or a string that can be used to get an SFX
+vol - The volume of the sound, excluding falloff
+vary - bool that determines if the sound changes pitch every time it plays
+extrarange - modifier for sound range
+falloff_exponent - Rate of falloff for the audio
+frequency - playback speed of audio
+channel - The channel the sound is played at
+pressure_affected - Whether or not difference in pressure affects the sound (E.g. if you can hear in space)
+ignore_walls - Whether or not the sound can pass through walls.
+falloff_distance - Distance at which falloff begins.
+
+*/
+
```

Comment:
You should describe how changing some of these vars effects the sound, like the higher the exponent, the steeper the falloff, etc. That way it's easier to use without understanding the underlying system perfectly

Reply:
this better?

Reply:
You should keep the above 1 thing